### DISCUSSION TITLE
Trouble Installing Security Onion ISO to Dell PowerEdge R740xd - Edit: Updating rpm on ISO?

### DISCUSSION THREAD

**Discussion by ccpaasmyser** [Security-Onion-Solutions/securityonion#7360] [Unsupported Versions]
So when I attempt to install the Security Onion ISO from the Webpage I get the following error:

"Unable to open input kickstart file: curl#37 - "Couldn't open file /tmp/part-include"

I've verified that the installation media I have burned the ISO to works, on another device, and it comes up to install just fine on that machine.

I've downloaded and successfully installed CentOS 7 on the machine I'm having issues with. I'd rather use the Security Onion ISO for a cleaner installation.

Any help would be greatly appreciated. We don't have any other linux systems in our environment which makes me the "linux" guy, and my knowledge is limited. I opted to buy the device directly from SecurtyOnion with the support and was turned down.

The system is a beast, has 128gb ram with 109-120 TB of storage depending on different RAID configurations we have tried.

**Comment by dougburks** [comment]
Is the machine configured for BIOS or UEFI?

**Reply by ccpaasmyser** [reply]
I've tried it both ways multiple times, even with different SATA operation Raid On or AHCI.

**Reply by dougburks** [reply]
Have you tried all the steps at https://docs.securityonion.net/en/2.3/trouble-booting.html?

**Reply by ccpaasmyser** [reply]
I have, and when i download Cent OS 7 basic iso it seems to install just fine. Not sure what the difference is with the SO ISO and the CentOS ISO.

**Reply by dougburks** [reply]
Here are some older issues and discussions that seem similar that might give you some additional clues:
https://github.com/Security-Onion-Solutions/securityonion/issues/1825
https://github.com/Security-Onion-Solutions/securityonion/discussions/1812
https://github.com/Security-Onion-Solutions/securityonion/issues/1819

We haven't had any reports like this in a while. I suspect it's something specific to your hardware and likely your RAID controller. Is it possible it's some kind of software RAID which requires a specific driver?

If all else fails, you can always install CentOS 7 and then add our components manually.

**Reply by ccpaasmyser** [reply]
Its a possibility that its a driver issue, i haven't fully tested all that yet. I'll look through your linked issues and see if they help. I was trying to avoid manually adding components as i'm not the best with Linux, i just know how to get around. In the hopes to have the most sound, solid, securty onion functionality. But, i may have to do it that way anyway

**Reply by ccpaasmyser** [reply]
i also see they released a new build, so i'm going to try that as well.

**Comment by ccpaasmyser** [comment]
Regarding this issue. I've just had my server admin guru take a look and this is what he is asking me to ask around for.

He is telling me that it appears that the Security Onion ISO doesn't have the perc drivers in the packages folder of the ISO that is needed to properly spin up the perc controller for this model of server. 

He asked if i could ask around and see if we could get the rpm that i've attached in the packages folder of the security onion iso so it will work with this brand of dell server. He got this rpm from dell directly.
[kmod-megaraid_sas-07.719.06.00_el8.4-1.x86_64.zip](https://github.com/Security-Onion-Solutions/securityonion/files/9354458/kmod-megaraid_sas-07.719.06.00_el8.4-1.x86_64.zip)

Any thoughts or input from anybody out there. I have looked around a handful of times on this hub and found similar discussions (https://github.com/Security-Onion-Solutions/securityonion/issues/1825) and even another anaylst's discussion here : https://github.com/Security-Onion-Solutions/securityonion/discussions/8492

Just trying to get it resolved without manually installing security onion overtop another distro.

**Reply by ccpaasmyser** [reply]
@dougburks any thoughts?

**Reply by isaacgolding** [reply]
I have a thought.  I don't know what Doug will or wont suggest...but If spinning up another distro and a manual install of SO doesn't sound appealing, you could look at using a VM.  Its not the best solution for some situations....but there are lots of SO users plugging away just fine in VM land.   I can also tell you that SO doesn't like my Dell 720 raid controller (perc) out of the box or even if I pass through from my VM and let SO see the controller.  It's not "the fix" but it is a workaround and in the long run might help you out more.

**Reply by ccpaasmyser** [reply]
We are trying to avoid doing that to lower the management overhead of the full system. We want to avoid updating the windows environment and whatever hypervisor software we use or any of that. just want one system. That is being considered if we cannot resolved the original issue.

**Reply by dougburks** [reply]
> @dougburks any thoughts?

We typically don't add drivers unless they are required for our Security Onion Solutions appliances.

**Reply by ccpaasmyser** [reply]
@dougburks any chance that we can get it spun up as a test?

**Reply by dougburks** [reply]
> @dougburks any chance that we can get it spun up as a test?

@ccpaasmyser No, as I mentioned in [my previous reply](https://github.com/Security-Onion-Solutions/securityonion/discussions/7360#discussioncomment-3426503), we typically don't add drivers unless they are required for our Security Onion Solutions appliances. You are welcome to spin your own custom ISO image if you would like. Also, as I mentioned in [an earlier reply](https://github.com/Security-Onion-Solutions/securityonion/discussions/7360#discussioncomment-2316494), if standard CentOS 7 installs then you can just add our components manually:
https://docs.securityonion.net/en/2.3/installation.html#installation-on-ubuntu-or-centos

**Reply by ccpaasmyser** [reply]
@dougburks Fair enough. just wanted to ask.

**Comment by reyesj2** [comment]
I have installed multiple different versions of Security Onion up-to .140 on R740xd's. What raid controller do you have? I have not had any issues with the H730 or H740.

**Reply by ccpaasmyser** [reply]
Looks like H750
